submissions/omawumiRebeccaMoju/HTML/BLK.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title><BLK></title>
+</head>
+<body>
+    <h1>BLK_ My Naturals</h1</a>
+<nav>
+    <div>
+        <ul>
+            <li>About Us</li>
+            <li>BLK_My Naturals</li>
+        </ul>
+    </div>
+</nav>
+<header>
+    <h2>About Us</h2>
+    <section>
+        <article>
+            <p>
+                BLK_My Naturals is a haircare brand that is all about wearing your natural kinky hair with pride.
+                Our products are made from strictly organic ingredients which helps give our naturals the healthy look it deserves. 
+                We have different products according to the different needs of the black natural hair.
+                BLK helps you wear your natural curls like the Queen you are. 
+            </p>
+        </article>
+    </section>
+</header>
+<header>
```

Comment:
You already used the header tag above. You can use main here instead of using the header tag again.